sos/policies/distros/__init__.py

```diff
@@ -71,19 +71,19 @@ class LinuxPolicy(Policy):
     def __init__(self, sysroot=None, init=None, probe_runtime=True):
         super(LinuxPolicy, self).__init__(sysroot=sysroot,
                                           probe_runtime=probe_runtime)
-        self.init_kernel_modules()
 
         # need to set _host_sysroot before PackageManager()
```

Comment:
This comment seems to get here by mistake, since PackageManager is initialized in child classes

Reply:
Yup, left over from moving the other bits around.

Reply:
Still (well, a nitpick) `s/set _host_sysroot/set sysroot/1` (or `set self.sysroot`)?

Reply:
I was just going to remove the comment entirely in my next update